eoscostarica/eosio-dashboard
# Fix/refresh contract table 789

### Change get Data button to Refresh Data

### What does this PR do?

- Resolves #858
- Resolves #789 
- Refactor code on AccountInfo, ContractActions, RicardianContract and ContractTables components
- Fix visual bug with horizontal scrollbar on contract table
- Debounce implementation on ContractTables

### Steps to test

1. Run the project locally
2. Go to the path of accounts 
3. Check the button

#### CheckList
